Ticket-booking-spring-app/src/main/java/com/example/demo/util/reservation_request/TicketType.java

```diff
@@ -0,0 +1,7 @@
+package com.example.demo.util.reservation_request;
+
+public enum TicketType {
+    ADULT,
+    STUDENT,
+    CHILD
```

Comment:
A co jeśli za jakiś czas kino uzna, że należy wprowadzić bilety dla emerytów? :)

Reply:
To możemy wtedy dopisać tutaj dodatkowy typ biletów :)
Starałam się tak to zaimplementować żeby sprawdzać i poprawność danych w bazie i typów biletów żądanych przez użytkownika.
Jak można to zrobić lepiej?